Tools/SeeDot/seedot/Predictor/library_fixed.cpp

```diff
@@ -954,6 +954,64 @@ void Maxpool(MYINT *A, MYINT *B, MYINT N, MYINT H, MYINT W, MYINT C, MYINT FH, M
 	return;
 }
 
+// A = Normalise(A)
+void NormaliseL2(MYINT* A, MYINT N, MYINT H, MYINT W, MYINT C, MYINT scaleA, MYINT shrA) {
```

Comment:
wasnt there an epsilon as well for division stability? could help if the vector is very close to zero